projects/storefrontlib/src/shared/components/checktable/checktable.component.html

```diff
@@ -0,0 +1,41 @@
+<ng-container *ngIf="tableData">
+  <table class="table cx-checktable">
+    <thead class="cx-thead-mobile">
+      <th scope="col" *ngFor="let column of columns">
+        {{ column.value }}
+      </th>
+    </thead>
+    <tbody>
+      <tr *ngFor="let row of tableData">
+        <td class="cx-code" *ngFor="let column of columns">
+          <!-- <div class="d-md-none cx-label" *ngIf="column.checked===false">
+            {{ column.value }}
+          </div> -->
+
+          <div *ngIf="column.checked">
```

Comment:
I think we can add hardcoded **td** before <td class="cx-code" *ngFor="let column of columns"> without any text, and also add Assign **th** before th listing